### PR TITLE
MFT: Async task morf to Track task and general json renaming

### PR DESCRIPTION
**Pull request by tomas-herman**
- Changing the Async task to Track task (renaming+removing the cluster plots)
- Deleting the old track task
- Renaming json files

### CODE REVIEW

**Comment by tomas-herman** [comment]
The changes from https://github.com/AliceO2Group/QualityControl/pull/1400 are contained in this PR.

File: Modules/MFT/src/QcMFTTrackCheck.cxx [hunk 1]
```diff
@@ -58,16 +58,16 @@ void QcMFTTrackCheck::beautify(std::shared_ptr<MonitorObject> mo, Quality checkR
     auto* histogram = dynamic_cast<TH1F*>(mo->getObject());
 
     if (checkResult == Quality::Good) {
```
Comment (Barthelemy) [inline Modules/MFT/src/QcMFTTrackCheck.cxx:60]: perhaps this whole block of code can be removed ?

**Review by Barthelemy — APPROVED** [review]

File: Modules/MFT/src/QcMFTTrackCheck.cxx (hunk 1 above)
Comment (tomas-herman) [reply Modules/MFT/src/QcMFTTrackCheck.cxx]: I removed it in the new commit

**Comment by tomas-herman** [comment]
This PR has to be deployed together with 

CW: https://github.com/AliceO2Group/ControlWorkflows/pull/540
O2DPG: https://github.com/AliceO2Group/O2DPG/pull/882

as these changes are interconnected. Also the automatic checks might fail due this reason.

**Review by knopers8 — APPROVED** [review]

**Pull request merged**